features/prefers-reduced-motion.yml

```
@@ -0,0 +1,12 @@
+name: prefers-reduced-motion media query
+description: The `prefers-reduced-motion` CSS media query sets styles based on whether the user prefers to minimize the amount of non-essential animations on the device.
+spec:
+  - https://drafts.csswg.org/mediaqueries-5/#prefers-reduced-motion
+  - https://wicg.github.io/user-preference-media-features-headers/#sec-ch-prefers-reduced-motion
```

I don't think the header spec really sheds any new light on this feature. Let's omit it.

```suggestion
spec: https://drafts.csswg.org/mediaqueries-5/#prefers-reduced-motion
```